src/Common/utils.ts

```diff
@@ -29,7 +30,7 @@ class Utils {
    * This function returns true for both X-addresses and classic addresses.
    * @see https://xrpaddress.info/
    *
-   * @param address An address to check.
+   * @param address - An address to check.
```

Comment:
This syntax (hyphen separating parameter name from description) for `@param` tags is enforced by the `tsdoc` ESLint plugin. I think this makes things a little more readable though.